rs/nervous_system/canisters/src/registry.rs

```diff
@@ -115,6 +116,12 @@ impl FakeRegistry {
         FakeRegistry::default()
     }
 
+    /// Test method to get the latest version in the FakeRegistry (as opposed to
+    /// get_latest_version which implements the Registry interface and returns a Result)
+    pub fn latest_version(&self) -> u64 {
```

Comment:
I don't see how this is a test-only function, neither the create, nor the `FakeRegistry` type, nor the function are marked as test-only. Can we enforce that this code is test-only?

Reply:
It's not actually important that you can't call it from other contexts, as the FakeRegistry is only intended for tests.  I should change the documentation.